t/cli/test_validate_config.sh

```diff
@@ -69,3 +69,23 @@ fi
 make stop
 
 echo "passed: allow configuring address in stream_proxy"
+
+sed -i 's/^  \(node_listen:\)/  #\1/g' conf/config-default.yaml
+sed -i 's/^    \(- 9080\)/    #\1/g' conf/config-default.yaml
+sed -i 's/^  # \(node_listen: 9080\)/  \1/g' conf/config-default.yaml
+
+echo '
+apisix:
+    node_listen:
```

Comment:
Could we keep the table in conf/config-default.yaml and use number in config.yaml? Better to avoid touching config-default.yaml in the test.

Reply:
I see.